robot_motion_planning_summary.tex

```diff
@@ -225,18 +255,122 @@ \subsection{Path and Map Smoothing}
 
 \section{Probabilistic Robotics}
 
-\td{noch nich fertig... \\ not yet written...}
+\td{noch nicht fertig... \\ not yet written...}
 \subsection{Kalman Filter}
 
+\paragraph{predict/ time update}
+
+\begin{align}
+	&\text{mean} && \hat{x}_{k+1}^- = A \hat{x}_{k} + B u_k  \\
+	&\text{covariance } \sigma_3^2 = \sigma_1^2 + \sigma_2^2 && P_{k+1}^- = A P_k A^\top + Q 
+\end{align}
+
+\paragraph{correct/ measurement update}
+
+\begin{align}
+	&\text{Kalman Gain} && K_k = P_k^- H^\top (H P_k^- H^\top + R)^{-1} \\
+	&\text{mean} && \hat{x}_k = \hat{x}_k^- + K_k (z_k - H \hat{x}_k^-) \\
+	&\text{covariance } \sigma_3^2 = \frac{\sigma_1^2  \sigma_2^2}{\sigma_1^2 + \sigma_2^2}  && P_k = (I - K_k H) P_k^-
```

Comment:
hm, I don't directly get it: what's the purpose of the expression with the sigmas here?

Reply:
This is just the merging rule of 2 normal distributions. Through the Kalman Gain it's a bit obfuscated.

Reply:
This is just the merging rule of 2 normal distributions. Through the Kalman Gain it's a bit obfuscated.

Reply:
I greyed it out so it's clear that it is just explanation.